packages/headless/src/chat.ts

```diff
@@ -9,6 +16,7 @@ import { MutationKey, QueryKey } from "@/constants";
 import { useMutation, useQuery, UseQueryResult } from "@tanstack/react-query";
 import { useSocket } from "@/socket";
 import { concat, uniqueId } from "lodash";
+import { ServerEvent, ServerEventPayload } from "@litespace/types/dist/esm/wss";
```

Comment:
You cannot import from `dist/esm`. I belive this from the auto-import in you IDE.